.wit01/CURRENT

```diff
@@ -0,0 +1 @@
+MANIFEST-000001
```

Comment:
Hey make sure to unstage this files!
Maybe it's time to add them to our gitignore file... 😋